File: Backend/src/Events.Domain/Panelist.cs
using System;
using System.Collections.Generic;

namespace Events.Domain
{
    public class Panelist
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string UrlImage { get; set; }
        public string Phone { get; set; }
        public int Email { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<PanelistEvent> PanelistsEvents { get; set; }
    }
}

File: Frontend/Events-App/src/app/model/Panelist.ts
import { SocialMedia } from './SocialMedia';
import { Event } from './Event'
export interface Panelist {
  id: number
  name: string
  description: string
  urlImage: string
  phone: string
  email: string
  socialMedias: SocialMedia[]
  panelistsEvents: Event[]
}
